src/main/java/graphql/util/LogKit.java

```diff
@@ -0,0 +1,20 @@
+package graphql.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LogKit {
```

Comment:
internal or public?

Reply:
internal and I think we need to reverse-the naming to 

notprivacysafe.graphql.xxx

so some one can do 

`notprivacysafe.graphql =  OFF` 

more easily